src/app/pages/Sandbox/Editor/Content/Header/ShareView.js

```diff
@@ -169,6 +193,7 @@ class ShareView extends React.PureComponent {
   setMixedView = () => this.setState({ showEditor: true, showPreview: true });
 
   setDefaultModule = id => this.setState({ defaultModule: id });
+  clearDefaultModule = () => this.setState({ defaultModule: null });
```

Comment:
I think you could add a check to the url generator like `if (defaultModule && defaultModule !== mainModule.shortid) { /* add the url thingie */ }`.